packages/rds/auth/secrets.ts

```diff
@@ -0,0 +1,17 @@
+import { SecretsManager } from "@aws-sdk/client-secrets-manager";
+const smClient = new SecretsManager({});
+
+export type DatabaseCredentials = {
+  username: string;
+  password: string;
+};
+
+export async function getAuthPassword(secretName: string): Promise<DatabaseCredentials> {
```

Comment:
Since we're returning a full object here rather than a password by itself:

```suggestion
export async function getDatabaseCredentials(secretName: string): Promise<DatabaseCredentials> {
```

Reply:
Refactored this and the `iam` strategy as well to use the same name and to return a `DatabaseCredentials` object.